opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -329,5 +340,42 @@ enum SpanFlags {
   // Bits 0-7 are used for trace flags.
   SPAN_FLAGS_TRACE_FLAGS_MASK = 0x000000FF;
 
-  // Bits 8-31 are reserved for future use.
+  // Bits 8 and 9 are used for the parent or link is remote flag.
+  // See SpanFlagsParentIsRemote and SpanFlagsLinkIsRemote for the
+  // valid values.
+  SPAN_FLAGS_PARENT_OR_LINK_IS_REMOTE_MASK = 0x00000300;
+
+  // Bits 10-31 are reserved for future use.
+}
+
+// SpanFlagsParentIsRemote represents constants used to interpret
+// bits 8 and 9 of the Span.flags field. These two bits represent whether
+// the span's parent is remote. Bit 8 represents whether the field has been
+// set and bit 9 represents whether the parent is remote.
+// Once the bits have been read using SPAN_FLAGS_PARENT_OR_LINK_IS_REMOTE_MASK,
+// the following constants can be used to interpret them.
+enum SpanFlagsParentIsRemote {
```

Comment:
```suggestion
enum NonW3cSpanFlags {
```
How about this? Or do we want one enum per flag group? Could also be sensible.

Reply:
I am not sure I like this approach. I think I prefer every group of bits to have its own enum. If we do what you suggest then future bits that we will carve out from the reserved bits will share the enum name and it will be easier to make a mistake and use the wrong flag.